Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.0"
+ruby "3.1.3"
```

Comment:
- [ ] [OPTIONAL] Please make sure the ruby version that you are using is the same here in the gemfile and also in the .ruby-version file, currently you have mentioned` ruby 3.1.3` in the gemfile and in your .ruby-version file it says `3.2.0`, consider keeping both of them same, it makes using your app easier for anybody who tries to clone it :+1: